Automattic/jetpack
# Social: Add recommendation steps for the V1 plan

Fixes https://github.com/Automattic/jetpack-reach/issues/439

## Proposed changes:

* Changed the advanced plan recommendation steps and cards to be used with the V1 plan

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/439
## Does this pull request change what data or activity we track or use?



## Testing instructions:







* Create a new JN site, or delete jetpack options on your local site to reset the onboarding flows. `jetpack docker wp option delete jetpack_options`. You'll have to reconnect Jetpack after this.
* Ensure you do not have the social plan, then buy it for the site. You should get redirected to the Social onboarding flow. There you should see 4 steps, the first one is a welcome, then SIG introduction, then a general monitor and accelerator pages.


https://github.com/Automattic/jetpack/assets/36671565/87efa715-d4ea-4556-98ef-30e1d776c798
